appleauth/auth_source.go

```diff
@@ -0,0 +1,153 @@
+package appleauth
+
+import (
+	"fmt"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/devportalservice"
+)
+
+// Source returns a specific kind (Apple ID/API Key) Apple authentication data from a specific source (Bitrise Service, manual input)
```

Comment:
Why do not we refer to `Bitrise Service` and `manual input` with the actual names `Bitrise Apple developer Connection` and `Step inputs`?

Reply:
fixed

Reply:
There are some leftover `Bitrise Service`:
- https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/pull/92/files#diff-b1ee36d4a84c1c9e5853aff27c0ab87c33d6ce9fbe2597ff126a071ac2c76488R17
- https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/pull/92/files#diff-b1ee36d4a84c1c9e5853aff27c0ab87c33d6ce9fbe2597ff126a071ac2c76488R23
- https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/pull/92/files#diff-84e3d7d3b6d0fc928e5f2bcceaca9f23d9cc2e3ca051569c5566f9414f6b45c8R44

Reply:
There is one more Bitrise Service and manual input: https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/pull/92/files#diff-84e3d7d3b6d0fc928e5f2bcceaca9f23d9cc2e3ca051569c5566f9414f6b45c8R46